client/cmd/dexc-desktop/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 replace decred.org/dcrdex => ../../..
 
 require (
-	decred.org/dcrdex v0.6.1
+	decred.org/dcrdex v0.6.0
```

Comment:
looks like an unintentional rollback

Reply:
Fixed! I guess this value doesn't really matter with the replace?

Reply:
Sorry to dwell on this, but all the indirect depends resulting from this are still changed.  Best just to restore these files from master e.g. `git restore --source=master go.mod`

Reply:
Ok! Will add that command to my knowledge.